src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/Utils.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace CSharpSyntaxGenerator
+{
+    internal static class Utils
+    {
+        public static bool IsTrue(string val)
```

Comment:
>IsTrue [](start = 27, length = 6)

Not sure we need a new file for this. A private method in AbstractFileWriter would seem sufficient.

Reply:
I use from my code :D   but i'm sure i could just reimplement ths.

Reply:
Btw, as a general comment, I think we still see Syntax.xml and the generator tools and APIs as internal. We still will not worry about incompatible breaks.

Exposing public APIs from this assembly seems counter to that, so I think we should avoid.

Reply:
sure.  will make this internal.  i still see this as internal as well.  by "my code" i mean "code that i intend to contribute to roslyn in another PR".  It would be in this project.  But it would want to do the same "truthy" check that the rest of this project does.  It's no biggie for me to copy that logic though for now :)

Reply:
Removed!